master/buildbot/process/builder.py

```diff
@@ -538,7 +538,8 @@ def getCollapseRequestsFn(self):
 
         return collapseRequests_fn
 
-    def _defaultCollapseRequestFn(self, master, builder, brdict1, brdict2):
+    @staticmethod
```

Comment:
we can also say s/Builder._defaultCollapseRequestFn/self._defaultCollapseRequestFn/ line 537